Header header
ColorBlobInfo3D[] color_blobs
